### DISCUSSION TITLE
Feedback on Braille Pc keyboard typing

### DISCUSSION THREAD

**Discussion by levilibegas** [nvaccess/nvda#14497] [Ideas]
### Is your feature request related to a problem? Please describe.
I would like to give 2 feedbacks on braille Pc Keyboard
### Describe the solution you'd like
I would like the Pc Braille keyboard development version be added at nvda Snapshots, so it could be easier to test (there are some changes not implemented in the stable addon that are very interesting to implement)
Second, I wonde if it would be doable to set each of the eight dots so as to suit any personneeds, for instance, setting it in vertical, or changig it mirrorwise, or so on.

I wonder if keys not used could be left as dead keys.
Thanks.

### Describe alternatives you've considered
I don't know if this Brl Keyboard could work along with braille extender addon.

### Additional context
I think that would make the Brl Keyboard addon absolutely flexible, letting configure several braille rows as needed so as to build an ergonomic keyboard by designing braille dots according to anyone's needs. Thanks.

**Comment by josephsl** [comment]
CC @NVDAEs

**Comment by seanbudd** [comment]
Hi @levilibegas.

NV Access does not develop this add-on. If you'd like to give feedback on the add-on, please contact the author.

We do not know how this add-on works or what it does. If you'd like to request specific features from the add-on, please create issues which outline the specific features you want in NVDA.

**Comment by levilibegas** [comment]
I am not susre then, is nvda braille pc keyboard an addon that is aswell integrated into nvda's core? I thought it was the case as I read the users guide. Thanks.

**Comment by nvdaes** [comment]
Hello:

I maintain the following add-on, originally developed by @jcsteh, who allowed me to maintain it. In case you want me to add some features, please create an issue on this repo  to discuss it:

https://github.com/nvdaes/pckbbrl

Hope this helps.
